tests/python/tenseal/test_enc_dec.py

```diff
@@ -94,3 +96,16 @@ def test_ckks_empty_encryption(plain_vec):
     with pytest.raises(ValueError) as e:
         ckks_vec = ts.ckks_vector(context, plain_vec, scale)
     assert str(e.value) == "Attempting to encrypt an empty vector"
+
+
```

Comment:
Could be possible to reuse the CKKSVector tests?

Reply:
I just wanted to test this simple functionality for now, and the CKKSVector tests aren't using plainvector objects and only takes vectors as input, which should be N-D for the CKKSTensor tests. So maybe they will need to be separate at some point.